docs/SDK/Explanation/authentication/sessionSigs.md

```diff
@@ -2,25 +2,132 @@
+## Background
 
-It works by generating a new random ed25519 keypair in the browser, and storing it in local storage. Then, the user signs a SIWE message with their wallet which contains the session public key. This signature is stored in local storage as well. The session keypair is used to sign all requests to the Lit Protocol API, and the user's wallet signature is sent along with the request, attached as a "capability" to the session signature. Each node in the Lit Network receives a unique signature for each request, and can verify that the user owns the wallet address that signed the capability.
+With Lit Protocol, the user needs to prove ownership of their wallet, and this is typically done via a wallet signature, sometimes referred to as AuthSigs in the Lit docs. However, in order to protect against replay attacks, to let users scope and delegate specific capabilities to specific resources, and to deliver a good user experience, we've implemented support for session signatures (enabled by session keys) during authentication.
 
-## Obtaining session signatures for a request in a browser
+## SessionSigs
 
-A prerequestive is that you must have a connected LitNodeClient and pass that into the getSessionSigs function.
+We refer to a session signature obtained from the user via session keys as a SessionSig.
 
-### For a signing request
+SessionSigs are produced by a ed25519 keypair that is generated randomly on the browser - these are stored in local storage. The first step to producing SessionSigs is to first obtain an AuthSig through one of the authentication methods outlined [here](/SDK/Explanation/authentication/methods). By specifying the session keypair's public key in the signature payload of the AuthSig, users can choose which specific actions to delegate to the session keypair for operating upon certain resources. This AuthSig is stored in local storage as well.
 
+The session keypair is used to sign all requests to the Lit Protocol API, and the user's AuthSig is sent along with the request, attached as a "capability" to the session signature. Each node in the Lit Network receives a unique signature for each request, and can verify that the user owns the wallet address that signed the capability.
+
+### An Improved User Experience
+
+The steps to obtain an AuthSig requires an interactive experience that involves manual steps from the end-user - whether clicking through the Google OAuth flow, placing their fingerprint on their platform authenticators, or clicking through their the respective modals of their externally-owned accounts (eg. MetaMask). These manual steps can be friction that drives users away from applications.
+
+On the other hand, the steps to obtain a SessionSig is completely non-interactive. The session keys and signature can all be done programmatically once an AuthSig has been obtained. 
```

Comment:
the "On the other hand" comparison seems to suggest that the end user may not be involved in obtaining a session sig, but the manual steps are still required to get the args for generating the session sigs.

and the "Format of SessionSigs" seems to imply that sessionSigs are really just signed authSigs.